playbooks/production/roles/webserver/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+vhosts:
+  - vhost_name: docs.kernelci.org
+    data_dir: /data/kernelci.org
+  - vhost_name: storage.kernelci.org
+    data_dir: /data/storage.kernelci.org
+  - vhost_name: storage.chromeos.kernelci.org
+    data_dir: /data/storage.chromeos.kernelci.org
```

Comment:
Also, could these be made a host var to allow reusing this role in staging/sandbox?

It could be a dict if we leave the contents as they are (`{vhost_name: data_dir}`) or just the list of names (`[vhost_name, ...]`) and another host var with main data dir for `Create data directories` task

Reply:
Yes, indeed might be useful for staging. I will think how to do that properly

Reply:
Created https://github.com/kernelci/kernelci-deploy/issues/193 to track it post-merge